Sync/Synchronizers/LoginsSynchronizer.swift

```diff
@@ -162,7 +162,9 @@ public class LoginsSynchronizer: IndependentRecordSynchronizer, Synchronizer {
             let lm = response.metadata.lastModifiedMilliseconds!
             log.debug("Applying incoming password records from response timestamped \(ts), last modified \(lm).")
             log.debug("Records header hint: \(response.metadata.records)")
-            return self.applyIncomingToStorage(logins, records: response.value, fetched: lm)
+            return self.applyIncomingToStorage(logins, records: response.value, fetched: lm) >>> effect {
+                NSNotificationCenter.defaultCenter().postNotificationName(NotificationDataLoginDidChange, object: nil)
```

Comment:
If you do this, you'll get a second sync immediately after the first, and if that adds records you'll get a third… and so on.